charts/Chart.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
     version: "1.18.0"
 name: marklogic
 description: MarkLogic Server is a multi-model database that has both NoSQL and trusted enterprise data management capabilities.
-appVersion: "11.1.0"
+appVersion: "11.2.0"
```

Comment:
Check list mentions that we need to update the Chart version to 1.1.2. Is it done differently ?